applications/tari_validator_node/src/p2p/services/epoch_manager/base_layer_epoch_manager.rs

```diff
@@ -247,7 +247,7 @@ impl BaseLayerEpochManager {
         // retrieve the validator nodes for this epoch from database
         let vns = self.get_validator_nodes_per_epoch(epoch)?;
 
-        let half_committee_size = 4; // total committee = 7
+        let half_committee_size = COMMITTEE_SIZE.div(2) + 1; // middle point of COMMITTEE_SIZE = 7
```

Comment:
If COMMITTEE_SIZE is even, it will be half + 1 .

```rust

        let half_committee_size = {
             // Ceiling div
             let v = COMMITTEE_SIZE / 2;
            let rem = COMMITTEE_SIZE % 2;
            if r > 0 {
                v + 1
            } else {
                v
            } 
};
```